libscript/src/system.lcb

```diff
@@ -56,6 +56,34 @@ begin
 	MCSystemExecGetOperatingSystem(output)
 end syntax
 
+public foreign handler MCSystemExecGetArchitecture(out Architecture as String) returns nothing binds to "<builtin>"
+/**
+Summary:    The architecture
+
+Example:
+    if the architecture is "x86_64" then
+        -- Do something x86_64 specific
+    end if
+
+Description:
+Returns a string describing the instruction set architecture
+that is being used by the machine LiveCode is running on.
+The possible values are:
+
+* "x86" - 32-bit Windows, Mac and Linux, and iOS simulator
+* "x86_64" - 64-bit Windows, Mac and Linux
```

Comment:
@livecodeali this should note iOS simulator too. I suspect this list could easily get out of date on linux and android without us noticing. Perhaps this should just be something like:

```
* "x86" - 32-bit x86 builds
* "x86_64" - 64-bit x86 builds
* "arm" - 32 bit arm builds
* "arm64" - 64-bit arm builds
* "" - Emscripten
```

Reply:
should emscripten be `js` the same as we use for platform id?

Reply:
oh yes, probably!